src/main/java/gov/cabinetofice/gapuserservice/util/WebUtil.java

```diff
@@ -0,0 +1,18 @@
+package gov.cabinetofice.gapuserservice.util;
+
+import jakarta.servlet.http.Cookie;
+
+public class WebUtil {
+
+    private WebUtil() {
+        throw new RuntimeException("Not allowed");
+    }
+
+    public static Cookie buildCookie(final Cookie cookie, final boolean isSecure, final boolean isHttpOnly) {
```

Comment:
Honestly not sure on this one, but would it make sense to have this as a bean so we don't run into issues trying to unit test another static dependency?

Reply:
Beans are more about being able to inject dependencies into classes while letting the framework manage creation and assignment of the object.

With unit tests we'd be fine to run the actual function instead of mocking it given how simple the logic is. But that's essentially your three choices with static code. 

1. Don't make things static 
2. Make them static and deal with difficulty mocking 
3. Make them static and run the real function during testing

Reply:
cool yeah that's reasonable. Agreed that option 3 here would be fine